R/model-tools.R

```diff
@@ -31,45 +31,28 @@
 #' enw_formula_as_data_list(prefix = "missing")
 enw_formula_as_data_list <- function(formula, prefix,
                                      drop_intercept = FALSE) {
-  paste_lab <- function(string, lab = prefix) {
-    paste0(lab, "_", string)
-  }
+  data <- list()
+  data[c("fintercept", "fdesign", "fnrow", "findex", "fnindex", "fncol", "rdesign", "rncol")] <- 0
+  data[c("fdesign", "fnrow", "findex", "fnindex", "fncol", "rdesign", "rncol")] <- list(numeric())
```

Comment:
this seems to be the incorrect collection of columns to assign to `numeric()` - copy-paste error?

Reply:
Also the wrong columns to 0 - this is on me, for not being clearer in the issue framing.